src/cast_core/include/cast_core/actors/OptionsConversion.hpp

```diff
@@ -0,0 +1,243 @@
+// Copyright 2019-present MongoDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef HEADER_1AFC7FF3_F491_452B_9805_18CAEDE4663D_INCLUDED
+#define HEADER_1AFC7FF3_F491_452B_9805_18CAEDE4663D_INCLUDED
+
+#include <string_view>
+
+#include <mongocxx/database.hpp>
+#include <mongocxx/pool.hpp>
+
+#include <gennylib/Actor.hpp>
+#include <gennylib/PhaseLoop.hpp>
+#include <gennylib/context.hpp>
+
+#include <value_generators/DocumentGenerator.hpp>
+
+namespace YAML {
+
+template <>
+struct convert<mongocxx::options::aggregate> {
```

Comment:
for my own sanity - did you add/change these at all or just straight up moving them here? I ask because I had to change them to `genny::NodeConvert` for https://github.com/mongodb/genny/pull/424. If I know what changes there were (if any) it'll make merging in these changes a lot easier.

Reply:
Only cut and paste, no changes. Although I do have changes in my next PR.